.github/workflows/makefile.yml

```diff
@@ -76,6 +39,6 @@ jobs:
         # When using ubuntu20.04
         BASE_IMG=ubuntu:20.04
         IMAGE_NAME=ipdk/p4-ovs-ubuntu20.04
-        DOCKERFILE=${SCRIPT_DIR}/../Dockerfile.ubuntu
+        DOCKERFILE=${GITHUB_WORKSPACE}/build/Dockerfile.ubuntu
```

Comment:
@mestery the problem here was that `${SCRIPT_DIR}` wasn't escaped, didn't exist and as such was replaced by "" i.e nothing.... Escaping should have also done the job like `\${SCRIPT_DIR}`. In the future this cat could be replaced by `./ipdk install ubuntu2004` when #80 has landed 😊